### ISSUE TITLE
BoxPlot Outliers

### ISSUE THREAD

**Issue by cpyfferoen** [apexcharts/Blazor-ApexCharts#555]
Apex Charts documentation shows using the Goals for Outliers - however, the ListPoint does not support Goals property.

**Comment by joadan** [comment]
Hi, 

Not sure I understand, do you mean goals like this https://apexcharts.github.io/Blazor-ApexCharts/bar-charts#goals

**Comment by cpyfferoen** [comment]
Yes exactly - Goals only exist on the DataPoint class - and do not exist on the ListPoint - which is the point leveraged by the ApexBoxPlotSeries
![image](https://github.com/user-attachments/assets/613a7bb6-2e39-430c-a6fb-8701119ee4c2)

**Comment by joadan** [comment]
Sorry but this is not supported in apexcharts.js or? 
If not you should add the enhancement request to the apexcharts.js repo.
